.github/workflows/pmem_test_matrix.yml

```diff
@@ -22,8 +22,8 @@ permissions: {}
 jobs:
   job:
     name: ${{ matrix.force_enable }}, ${{ matrix.test_script }}, ${{ matrix.os }}, ${{ matrix.build }}
-    if: github.repository == 'pmem/pmdk'
-    runs-on: [self-hosted, "${{ matrix.os }}"]
+    if: github.repository == 'daos-stack/pmdk'
+    runs-on: [self-hosted, pmem, "${{ matrix.os }}"]
```

Comment:
I understand `pmem` is a label for the runners equipped with the priceless piece of equipment. Are they assigned solely to the PMDK validation or are they available for other repositories as well? I just though it might be better to call them `pmdk` if they are solely for this purpose. Just a though.

Reply:
+1

Reply:
Done